recipes/slurmpter/meta.yaml

```diff
@@ -17,11 +17,11 @@ build:
 requirements:
   host:
     - pip
-    - python
+    - python>=3.5
```

Comment:
You need a space between i.e `python >3.5` both here and under `run`

Reply:
FIxed.